src/toil/common.py

```diff
@@ -413,6 +414,10 @@ def _addOptionFn(*name, **kwargs):
                       "missing preemptable nodes with a non-preemptable one. A value of 1.0 "
                       "replaces every missing pre-emptable node with a non-preemptable one." %
                       config.preemptableCompensation))
+    addOptionFn("--workerDiskSize", dest="workerRootVolSize", default=None,
+                help=("Specify the size of the root volume of worker nodes when they are launched "
+                      "in gigabytes. You may want to set this if your jobs require a lot of disk "
+                      "space. The default value is 50."))  # Note: the default is set in _getBlockDeviceMapping
```

Comment:
Yes. The only reason I didn't was because then the "optional" function argument is always being passed which seemed a little silly to me. But Since this is more clear to the user then I suppose it makes more sense.